src/runtime/pay-flow.js

```diff
@@ -71,21 +80,37 @@ export class PayStartFlow {
    * @return {!Promise}
    */
   start() {
-    // Start/cancel events.
-    this.deps_.callbacks().triggerFlowStarted(SubscriptionFlows.SUBSCRIBE, {
+    const triggerFlowKeys = {
       'sku': this.sku_,
-    });
+    };
+
+    const swgKeys = {
+      'publicationId': this.pageConfig_.getPublicationId(),
+      'skuId': this.sku_,
+    };
+
+    if (!!this.oldSku_) {
+      triggerFlowKeys['oldSku'] = this.oldSku_;
```

Comment:
This is good, but to clarify the priority: payment payload and trigger params are very different - the payment payload travels across several layers to in a set of separate system stacks. That's why I prefer to keep that payload as clean as possible. Trigger parameters are different from this - it's all ours.

Reply:
ack